changelog/1.28.1.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "1.28.1"
+description: "Released on 02/23/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.28.1.
+
+### Features ✨
+
+There are no new features in 1.28.1.
```

Comment:
Should mention that the k8s requirement is relaxed now:
- We hard require 1.19+
- We warn on <= 1.20